test/cypress/e2e/ex-protractor-cucumber/followedAccount.spec.js

```diff
@@ -0,0 +1,25 @@
+import accounts from '../../../constants/accounts';
+
+describe('Followed accounts', () => {
+  it('should add a followed account to list', () => {
+    cy.loginUI(accounts.genesis, 'dev');
```

Comment:
not sure if it's worth adding some comments along the statements, indicating briefly what's going on on them.

```
// click add follow account
...
// fill in account details
...
// verify account added
...
// remove followed account
```

Reply:
I was also thinking about how to be more descriptive about what the steps are doing, but comments IMO will be a nightmare to keep up-to-date.
Maybe if we keep the comments on the same line as the command, so that it doesn't happen that you paste something in between:
```
    cy.get('.add-account-button').click(); // click add follow account
    cy.get('.address input').click().type('94495548317450502L'); // fill in account details
    cy.get('.next').click(); // go to next step
    cy.get('.next').click(); // go to next step
```
But does it really add so much information?

Reply:
I think the text description above test scenario should be provided in some cases. But future tests should be granular having name self-explaining what happens there with no intricant actions inside